remake.yml

```diff
@@ -15,20 +15,17 @@ targets:
   all:
     depends: 3_visualize/out/figure_1.png
 
-  site_data:
-    command: download_nwis_data()
+  download_files:
+    command: download_nwis_data(dl_dir = '1_fetch/out')
```

Comment:
You are probably getting a warning on the "1_fetch/out" here that doesn't make sense (yet)
```
In addition: Warning messages:
1: In structure(.Call(C_Rmd5, files), names = files) :
  md5 failed on file '1_fetch/out'
```
We'll cover that in an upcoming section. But you'll see a different way to handle this in the future sections.